.gitignore

```diff
@@ -19,4 +18,4 @@ zip_info.txt
 .cache/*
 !tests/testdata/journal.pbio.2001413.xml
 !tests/testdata/plos.correction.3155a3e9-5fbe-435c-a07a-e9a4846ec0b6.xml
-!tests/testdata/journal.pone.0185809.xml
\ No newline at end of file
+!tests/testdata/journal.pone.0185809.xml
```

Comment:
add `dois.txt` to `.gitignore`
then add the text file to repo

Reply:
if dois.txt is part of the project why should be in the .gitignore file?

Reply:
sorry, I mean add '!dois.txt' in case it was being blocked by gitignore

Reply:
will do it

Reply:
funny thing that I will have to do the dois.txt file again since I deleted all PLOS related data from my computer because I thought I commited everything, but didn't know at that time that the dois file was ignored.